jdd.css

```diff
@@ -254,6 +254,7 @@ pre.error {
     border-radius: 4px;
     padding: 1em;
     display: none;
+    text-wrap: wrap;
```

Comment:
It seems that you changed the indentation character(s) here. Compare the line above.